src/request/dispatch.ts

```diff
@@ -1,14 +1,20 @@
 import { Ctx } from '../ctx'
 import { postHandler } from './post'
 
+function opFromPath(url: string): string {
+  const res = new RegExp('\\/([^\\/]*)', 'g').exec(new URL(url).pathname)
```

Comment:
does this regex work? i get an error for it on regex101 on javascript mode

Reply:
```suggestion
  const res = new RegExp('^\\/([^\\/]*)').exec(new URL(url).pathname)
```

@thedavidmeister - [it works for me](https://regex101.com/r/0926nk/1) :shrug: - don't forget to un-double the backslashes due to the string processing... I couldn't use a regex literal because of a bug in the typescript prettier linter.

But, while I was testing that, it occurred to me I shouldn't be using the global flag, since I only need one result, and I could put in the start marker to shore it up a bit.

Reply:
ooh yeah the double backslash is what i was looking at